Cargo.toml

```diff
@@ -134,11 +134,12 @@ branch = "main"
 package = "bson"
 version = "3.0.0"
 optional = true
+features = ["serde"]
 
 [dependencies.mongocrypt]
-git = "https://github.com/mongodb/libmongocrypt-rust.git"
-branch = "main"
-version = "0.3.0"
+git = "https://github.com/abr-egn/libmongocrypt-rust.git"
```

Comment:
I'll revert this once https://github.com/mongodb/libmongocrypt-rust/pull/45 is merged.

Reply:
Note: this is also the cause of the `cargo deny` failure.